linux_mode/raw2dmp/.gitignore

```diff
@@ -0,0 +1 @@
+raw2dmp
```

Comment:
This change should be gone once `raw2dump`'s logic is merged into the Python script.